inst/book/doublet-detection.Rmd

```diff
@@ -201,15 +202,13 @@ We can also extract explicit doublet calls for each cell based on the final thre
 table(sce.mam.dbl$scDblFinder.class)
 ```
 
-Compared to `computeDoubletDensity()`, `scDblFinder()` provides a more sophisticated approach that makes greater use of the information in the simulated doublets.
-This can improve performance of doublet calling in difficult scenarios, e.g., if many within-cluster doublets are present.
-However, this strategy has a slightly stronger reliance on clustering - and thus sensitivity to the vagaries of cluster formation - 
-as this is used to compute some of the metrics mentioned above.
+Compared to `computeDoubletDensity()`, `scDblFinder()` provides a more sophisticated approach that makes greater use of the information in the simulated doublets and can improve the accuracy of doublet identification in difficult scenarios.
```

Comment:
Break the sentence at the "and" into a new sentence. I find long sentences difficult to parse, and also, you can fit more information in if you have another sentence to make your point. Something like:

> This can improve the accuracy of doublet identification in difficult scenarios, e.g., where doublets form between cell types with highly variable total RNA content.

Don't know if that is one such difficult scenario where accuracy is improved, but you could probably come up with a very brief mention-in-passing of a relevant difficult scenario.

Reply:
Tried to add examples in 91cbf2a , avoiding committing to the differences in contributions being due to original RNA content.